base/utils/atomic.go

```diff
@@ -40,7 +43,12 @@ func CreateAtomic(dest string, r io.Reader, opts *AtomicFileOptions) error {
 	defer tmpFile.Cleanup() //nolint:errcheck
 
 	if opts.Mode != 0 {
-		if err := tmpFile.Chmod(opts.Mode); err != nil {
+		if runtime.GOOS == "windows" {
+			err = acl.Chmod(filepath.Join(opts.TempDir, dest), opts.Mode)
```

Comment:
Did you really mean to change the permission of the directory and not the permissions of the file?